library/pod-security-policy/seccomp/src_test.rego

```diff
@@ -0,0 +1,75 @@
+package k8spspseccomp
+
+test_input_seccomp_allowed_all {
+    input := { "review": input_review, "parameters": input_parameters_wildcard}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_allowed_in_list {
+    input := { "review": input_review, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_not_allowed_not_in_list {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) > 0
+}
+
+test_input_seccomp_not_allowed_no_annotation {
+    input := { "review": input_review_no_annotation, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) > 0
```

Comment:
Can you pls link to the doc where it says `seccomp.security.alpha.kubernetes.io/pod` annotation must be provided?

Reply:
Kubernetes docs for these are not great. it's mentioned here: https://docs.openshift.com/container-platform/3.5/admin_guide/seccomp.html#overview

> seccomp.security.alpha.kubernetes.io/pod: profile applies to all containers in the pod that do not override